core/src/main/scala/io/qbeast/model/CubeWeights.scala

```diff
@@ -34,18 +34,24 @@ class CubeWeightsBuilder(
     this
   }
 
+  // TODO add more logic and set at configurable parameter
+  def estimatePartitionCubeSize: Double = {
+    if (desiredSize.toDouble / numPartitions < 1000) {
```

Comment:
Yes, understand the global configuration on resources. 
But, in the case these number is used, it will affect the `cubeSize` specified by the user. Should fail if not reached, no?

Reply:
if cubeSize < minimumParititionDesiredSize it should either give a warning. I think it is enough, because one can want to run a jar where they set up a specific cubeSize in a cluster managed by another party that has configured the minumumPartitionDesired... You should get a warning, but you are not losing data, nor anything bad so it should just warn you.